recipes/automake/all/conanfile.py

```diff
@@ -34,107 +32,91 @@ def configure(self):
-
-        bin_path = os.path.join(self.package_folder, "bin")
-        self.output.info("Appending PATH environment variable:: {}".format(bin_path))
-        self.env_info.PATH.append(bin_path)
-
-        bin_ext = ".exe" if self.settings.os == "Windows" else ""
-
-        aclocal = tools.unix_path(os.path.join(self.package_folder, "bin", "aclocal" + bin_ext))
-        self.output.info("Appending ACLOCAL environment variable with: {}".format(aclocal))
-        self.env_info.ACLOCAL.append(aclocal)
-
-        automake_datadir = tools.unix_path(self._datarootdir)
-        self.output.info("Setting AUTOMAKE_DATADIR to {}".format(automake_datadir))
-        self.env_info.AUTOMAKE_DATADIR = automake_datadir
+            # tracing using m4 on Windows returns Windows paths => use cygpath to convert to unix paths
+            replace_in_file(self, path.join(self.source_path, "bin", "aclocal.in"),
+                            "          $map_traced_defs{$arg1} = $file;",
+                            "          $file = `cygpath -u $file`;\n"
+                            "          $file =~ s/^\\s+|\\s+$//g;\n"
+                            "          $map_traced_defs{$arg1} = $file;")
 
-        automake_libdir = tools.unix_path(self._automake_libdir)
-        self.output.info("Setting AUTOMAKE_LIBDIR to {}".format(automake_libdir))
-        self.env_info.AUTOMAKE_LIBDIR = automake_libdir
+        _ = self._autotools()
 
-        automake_perllibdir = tools.unix_path(self._automake_libdir)
-        self.output.info("Setting AUTOMAKE_PERLLIBDIR to {}".format(automake_perllibdir))
-        self.env_info.AUTOMAKE_PERLLIBDIR = automake_perllibdir
+    def package(self):
+        autotools = self._autotools()
+        # KB-H013 we're packaging an application, place everything under bin
+        autotools.install(args=[f"DESTDIR={unix_path(self, path.join(self.package_folder, 'bin'))}"])
 
-        automake = tools.unix_path(os.path.join(self.package_folder, "bin", "automake" + bin_ext))
-        self.output.info("Setting AUTOMAKE to {}".format(automake))
-        self.env_info.AUTOMAKE = automake
+        copy(self, "COPYING*", src=self.source_folder, dst=path.join(self.package_folder, "licenses"))
+        rmdir(self, path.join(self._datarootdir, "info"))
+        rmdir(self, path.join(self._datarootdir, "man"))
+        rmdir(self, path.join(self._datarootdir, "doc"))
 
-        self.output.info("Append M4 include directories to AUTOMAKE_CONAN_INCLUDES environment variable")
+    def _set_env(self, var_name, var_path):
+        self.output.info(f"Setting {var_name} to {var_path}")
+        self.buildenv_info.define_path(var_name, var_path)
+        setattr(self.env_info, var_name, var_path)
 
-        self.user_info.compile = os.path.join(self._automake_libdir, "compile")
-        self.user_info.ar_lib = os.path.join(self._automake_libdir, "ar-lib")
+    def package_info(self):
+        # KB-H013 we're packaging an application, hence the nested bin
```

Comment:
not sure comment is useful, maintainers know that

Reply:
The audience is more then just maintainers. You also have semi layman maintainers such as I, with just enough knowledge to mess things up. See the first attempt of my Autoconf PR, were I treated that recipe like a regular run recipe and not a build application.

I just recently came to this knowledge, because of a linter warning in the PR for my Autoconf recipe. while I have been wondering for over half a year what was "wrong" with the cpython recipe because everything was placed in the the `bin` dir for that as well.

Just give the word if the comment does more harm then good and I will remove it.

Reply:
The comment is wrong, there is no reason to nest bin folder.

Reply:
is the comment wrong or the linting warning?